packages/zoe/src/zoeService/zoe.js

```diff
@@ -131,11 +136,7 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
       // The issuers may not have been seen before, so we must wait for the
       // issuer records to be available synchronously
       const issuerRecords = await initIssuers(issuerPs);
```

Comment:
I notice that both zoe.js and contractFacet.js have identical `initIssuers` functions. Can we avoid this redundancy?

Reply:
Yup, that will be done in an upcoming PR :)